src/index.js

```diff
@@ -57,40 +51,48 @@ const BUTTONS_ARRAY = [
         const headerElement = document.querySelector("header")
         const addNumberElement = document.querySelector("p")
         addNumberElement.innerText = "Add Number"
-
-        const handleDisplayElement = () => {
-            const displayElement = document.getElementById("display")
-            console.log(displayElement)
-        }
+        addNumberElement.style.visibility = "hidden"
 
         // MAIN
-
         const mainElement = document.querySelector("main")
+        const callDeleteWrapperElement = document.getElementById("call-delete-wrapper")
+        const callButtonElement = document.getElementById("call-button")
+        callButtonElement.style.visibility = "hidden"
+        const deleteButtonElement = document.getElementById("delete-button")
+        deleteButtonElement.style.visibility = "hidden"
+
         const numbersToDisplay = []
 
         BUTTONS_ARRAY.forEach((button, index) => {
-            const handleDialButtonOnClick = () => {
-                numbersToDisplay.push(buttonValueElement.innerHTML)
-                console.log(numbersToDisplay)
+            const handleDisplayElement = () => {
+                const displayElement = document.getElementById("display")
```

Comment:
There is no need to define this element here as this will never change - it would be better to place it in the beginning of your main function.